engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/command/AttachVisualizationCmd.scala

```diff
@@ -69,17 +41,6 @@ class AttachVisualizationCmd(
     }
   }
 
-  private def replyWithContextNotExistError()(implicit
-    ctx: RuntimeContext,
-    ec: ExecutionContext
-  ): Future[Unit] = {
-    Future {
-      reply(
-        Api.ContextNotExistError(request.visualizationConfig.executionContextId)
-      )
-    }
-  }
-
   override def toString: String = {
     "AttachVisualizationCmd(visualizationId: " + request.visualizationId + ",expressionId=" + request.expressionId + ")"
```

Comment:
It should be possible to move `toString` into `ContextCmd` and use `getClass().getSimpleName()` concatenated with the common remainder of the text. Long live _deep inheritance hierarchy_! Not my favorite, but better than copy paste programming, indeed.